ENIGMAsystem/SHELL/Universal_System/estring.cpp

```diff
@@ -242,13 +244,7 @@ string string_replace(string str,string substr,string newstr) {
 }
 
 string string_replace_all(string str,string substr,string newstr) {
-  size_t pos = 0;
-  const size_t sublen = substr.length(), newlen = newstr.length();
-    while((pos=str.find(substr,pos)) != string::npos) {
-    str.replace(pos,sublen,newstr);
-    pos += newlen;
-  }
-  return str;
+  return ::string_replace_all(str, substr, newstr);
```

Comment:
ew but okay

Reply:
I wasn't sure if I could pull it into user namespace with a using or something without jdi exploding